### ISSUE TITLE
Regression ---- he Vsvim repo has noticed that building -standalone yields an FSharp.Core reference

### ISSUE THREAD

**Issue by KevinRansom** [dotnet/fsharp#13345]
The Vsvim repo has noticed that building -standalone yields an FSharp.Core reference

I have reproed this on my machine:

repro:
1.   Clone vsvim
2. build
3. test
4. observe failure (below)

````
  Starting:    Vim.VisualStudio.Shared.2019.UnitTest
    Vim.VisualStudio.UnitTest.CodeHygieneTest.FSharpCoreReferences [FAIL]
      Assert.NotEqual() Failure
      Expected: Not "FSharp.Core"
      Actual:   "FSharp.Core"
      Stack Trace:
        Test\VsVimSharedTest\CodeHygieneTest.cs(75,0): at Vim.VisualStudio.UnitTest.CodeHygieneTest.FSharpCoreReferences()

````

**Comment by jaredpar** [comment]
Given that this was moved to July does that mean that VS 17.2 will not be patched with a fix? Basically that we need to wait until 17.3 is released before there is a fix for this? 

That is dissapointing if true because I'm basically blocked from releasing new versions of VsVim until this is fixed.

**Comment by vzarytovskii** [comment]
I don't think we will make it to the servicing unfortunately, we should probably take it into next 17.3 preview. I know @KevinRansom wanted to look at it.
